### PR TITLE
Add e2e test for trigger warning in trigger state

### PR DESCRIPTION
**Pull request by Piotrk39**
## Describe your changes

Adds test that checks for amount of triggers (of any type) and compares it with the number of aggregated events.

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review

### CODE REVIEW

File: tests/e2e/tests/DisasterType/SwitchBetweenDisasterType.spec.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ test.beforeEach(async ({ page }) => {
   );
 });
 
-test(
+test.skip(
```
Comment (jannisvisser) [inline tests/e2e/tests/DisasterType/SwitchBetweenDisasterType.spec.ts:36]: please add a reason for skipping and an item for fixing it.

File: tests/e2e/tests/ChatSection/TriggerStateShouldHaveWarningEvents.spec.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      lastName: TriggerDataSet.lastName,
+    });
+    await chat.allChatButtonsArePresent();
+
+    // get the number of warning events and aggregated events
+    const warningCount = await chat.chatPredictionButtonsAreActive();
+    const aggregatesNumber = await aggregates.getNumberOfPredictedEvents();
+
+    // check if the number of warning events is equal to the number of aggregated events
+    expect(warningCount).toEqual(aggregatesNumber);
```
Comment (jannisvisser) [inline tests/e2e/tests/ChatSection/TriggerStateShouldHaveWarningEvents.spec.ts:63]: I'm not really getting this. I'm thinking maybe you're not fully clear on the terminology of 'trigger' (event above trigger) vs 'warning' (event below trigger threshold) vs 'alert' (this groups trigger+warning). Let's talk in person about this.

File: tests/e2e/tests/ChatSection/TriggerStateShouldHaveWarningEvents.spec.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    });
+    await chat.chatColumnIsVisibleForTriggerState({
+      firstName: TriggerDataSet.firstName,
+      lastName: TriggerDataSet.lastName,
+    });
+    await chat.allChatButtonsArePresent();
+
+    // get the number of warning events and aggregated events
+    const warningCount = await chat.chatPredictionButtonsAreActive();
+    const aggregatesNumber = await aggregates.getNumberOfPredictedEvents();
```
Comment (jannisvisser) [inline tests/e2e/tests/ChatSection/TriggerStateShouldHaveWarningEvents.spec.ts:60]: I think it's good to connect nr of events in chat with nr of events in aggregates section, but that is something else then what the test description above says. That doesn't talk about aggregates section.

File: tests/e2e/tests/ChatSection/TriggerStateShouldHaveWarningEvents.spec.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    accessToken,
+  );
+
+  await page.goto('/');
+  await loginPage.login(TriggerDataSet.UserMail, TriggerDataSet.UserPassword);
+});
+
+test(
+  qase(
+    43,
```
Comment (jannisvisser) [inline tests/e2e/tests/ChatSection/TriggerStateShouldHaveWarningEvents.spec.ts:37]: no written out test case in Qase

**Review by jannisvisser — CHANGES_REQUESTED** [review]

File: tests/e2e/tests/ChatSection/TriggerStateShouldHaveWarningEvents.spec.ts (hunk 2 above)
Comment (jannisvisser) [reply tests/e2e/tests/ChatSection/TriggerStateShouldHaveWarningEvents.spec.ts:63]: call this 'chatEventCount' vs 'aggregatesEventCount'

File: tests/e2e/tests/ChatSection/TriggerStateShouldHaveWarningEvents.spec.ts (hunk 3 above)
Comment (Piotrk39) [reply tests/e2e/tests/ChatSection/TriggerStateShouldHaveWarningEvents.spec.ts]: Updated description.

File: tests/e2e/tests/ChatSection/TriggerStateShouldHaveWarningEvents.spec.ts (hunk 2 above)
Comment (Piotrk39) [reply tests/e2e/tests/ChatSection/TriggerStateShouldHaveWarningEvents.spec.ts]: Changed!

File: tests/e2e/tests/DisasterType/SwitchBetweenDisasterType.spec.ts (hunk 1 above)
Comment (Piotrk39) [reply tests/e2e/tests/DisasterType/SwitchBetweenDisasterType.spec.ts:37]: Done!

**Review by jannisvisser — APPROVED** [review]

**Pull request merged**